geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreTest.scala

```diff
@@ -8,10 +8,11 @@
 
 package org.locationtech.geomesa.accumulo.data
 
-import java.io.IOException
+import java.io.{File, IOException}
```

Comment:
Oops. Yeah that hung around from when I tried reading example conf files for tests